### DISCUSSION TITLE
Java update

### DISCUSSION THREAD

**Discussion by Bmartin3k** [AlexProgrammerDE/balena-minecraft-server#134] [Q&A]
I've been running the server for months and works great. I do have one question.
The new 1.17 should be coming out soon, and it requires openJRE 16, and the server is currently running openJRE 11. I've tried to manually update but after a restart it goes back to 11. How do I update java on mc-server. Thanks in advance.

Blake

**Comment by viedok** [comment]
i am also waiting for this to update my server :D

**Reply by prismhour** [reply]
Same here..
